composer.json

```diff
@@ -62,7 +62,7 @@
   },
   "extra": {
     "branch-alias": {
-      "dev-master": "1.0.x-dev"
+      "dev-master": "develop"
```

Comment:
Sure we want to use `develop` for this, and not `master`?

Reply:
Good idea. Changed this.